packages/apputils/src/vdom.ts

```diff
@@ -127,6 +160,95 @@ export class ReactElementWidget extends VDomRenderer<any> {
   private _es: Array<React.ReactElement<any>> | React.ReactElement<any> | null;
 }
 
+/**
+ * Props for the UseSignal component
+ */
+export interface IUseSignalProps<SENDER, ARGS> {
+  // Phosphor signal to connect to
```

Comment:
These should be doc comments so they are picked up by automated tools as documentation:

```
/**
 * Phosphor signal to connect to
 */
```

Reply:
Ah thanks.